VMBackup/main/Utils/EventLoggerUtil.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, event_directory, severity_level):
         self.event_logging_error_count = 0
         self.events_folder = event_directory
         self.event_logging_enabled = bool(self.events_folder)
+        self.use_threads = use_async_event_logging
```

Comment:
let us call this variable async_event_logging locally too instead of use_threads?